docs/book/releases/v2.0.0.md

```diff
@@ -0,0 +1,94 @@
+<!-- markdownlint-disable MD034 -->
+# vSphere CSI Driver - v2.0.0 release
+
+## New Feature
+
+- Offline Persistent Volume expansion for Block Volume.
+- ReadWriteMany volumes using vSAN file services.
+- Support for enabling leader election to ensure High Availability for vSphere CSI driver.
+
+Note: This release is not backward compatible to vSphere 67u3 release.
+
+## Notable Changes
+
+- Support for enabling leader election to ensure High Availability for vSphere CSI driver.
+- Enhanced driver to ensure volume operations are idempotent.
+- Enhanced driver logging and debuggability with contextual log tracing.
+
+## Deployment files
+
+- https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/manifests/v2.0.0
+
+## Minimum Kubernetes Version
+
+- v1.16.0
+
+## Supported sidecar containers versions
+
+- csi-provisioner - v1.4.0
+- csi-attacher - v2.0.0
+- csi-resizer - v0.3.0
+- livenessprob - v1.1.0
+- csi-node-driver-registrar - v1.2.0
+
+## Known Issues
+
+### vSphere CSI Driver issues
+
+1. Unused volumes not deleted during full sync when CSI driver v2.0.0 is used with vSphere 6.7 Update3.
+   - Impact: Full sync in CSI v2.0.0 does not delete unused volumes in vSphere 6.7 Update3.
+   - Workaround: If you are using vSphere 6.7 Update3, it is recommended to use v2.0.1 release of the driver.
```

Comment:
If this is the reason why you are saying it is not backwards compatible, we should reword that statement.

Reply:
yes